src/components/Link.tsx

```diff
@@ -66,6 +66,7 @@ const GlossaryIcon = styled(Icon)`
 export interface IProps {
   to?: string
   href?: string
+  title?: string
```

Comment:
Why add `title` attribute out of curiosity?

Reply:
Probably this was not the correct solution.

The problem was that for example, in the `eth.tsx` page, we have this:
```tsx
<ButtonLink to="/get-eth/" title="where to buy eth">
```

That ButtonLink is going to be a Link underneath.

Help me with the thinking here, in this case, ButtonLink shouldn't send a `title` prop, right? because it is going to be a Link. The `title` only make sense in a `button` tag.

Reply:
Ah, I see. Ya given that `ButtonLink` isn't actually a button, no tooltip is generated, so I don't think it needs a `title` prop. I'd say let's remove that from those instances.